diff --git a/scripts/build.sh b/scripts/build.sh
index 5ff399039..5a5946f3e 100755
--- a/scripts/build.sh
+++ b/scripts/build.sh
@@ -2,8 +2,8 @@

 # This script builds the application from source for multiple platforms.
 # Determine the arch/os combos we're building for
-ALL_XC_ARCH="386 amd64 arm arm64 ppc64le mips mips64 mipsle mipsle64 s390x"
-ALL_XC_OS="linux darwin windows freebsd openbsd solaris"
+ALL_XC_ARCH="amd64"
+ALL_XC_OS="linux"
 SKIPPED_OSARCH="!darwin/arm !darwin/arm64 !freebsd/arm !freebsd/arm64"

 # Exit immediately if a command fails

